root/static/scripts/common/i18n/expand2.js

```diff
@@ -167,11 +167,11 @@ export function parseContinuous<T, U, V>(
 
 function concatStringMatch(
   accum: string | NO_MATCH,
-  match: string | NO_MATCH,
+  match: string,
 ): string {
   return (
     (gotMatch(accum) ? accum : '') +
-    (gotMatch(match) ? match : '')
```

Comment:
What's the change here? (since this file is seeing a bunch of changes that are not just type improvements, it should probably be mentioned somewhere :) )

Reply:
I thought it was related to the 0.217.0 upgrade but you're right, it's not. I moved it to a separate commit. :)